playwright/tests/rex-test/rex.behaviorspec.ts

```diff
@@ -223,15 +223,53 @@ test('note in content page', async ({ page, isMobile }) => {
 
   // AND: Add note to the highlight and save
   const noteText = randomstring()
-  await BookPage.clickHighlight(0)
+  const highlightId = await BookPage.highlight_id(randomparanumber)
+  await BookPage.clickHighlight(highlightId)
   await BookPage.addNote(noteText)
   await BookPage.noteConfirmDialog(Actions.Save)
 
-  // AND: Edit note to the highlight and save
-  const editnoteText = randomstring()
-  await BookPage.clickContextMenu(0)
-  await BookPage.editHighlight()
-  await BookPage.editNote(editnoteText)
+  // // AND: Edit note to the highlight and save - Edit Note Update in next PR
+  // const editnoteText = randomstring()
+  // await BookPage.clickContextMenu(0)
+  // await BookPage.editHighlight()
+  // await BookPage.editNote(editnoteText)
+  // await BookPage.noteConfirmDialog(Actions.Save)
+  // expect(await BookPage.noteText()).toBe(editnoteText + noteText)
+})
+
```

Comment:
Do you need this comment?

Reply:
I am going to update these lines in the next PR. So I prefer this staying here commented.